### PR TITLE
New Ollama Query Engine Pack

### PR DESCRIPTION
**Pull request by chnsagitchen**
# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

Fixes # 677

## Type of Change

Please delete options that are not relevant.

- [ ] New Pack

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] I stared at the code and made sure it makes sense
- [ ] I integrated with local ollama and llama2 model for the implementation of the new query engine

# Suggested Checklist:

- [ ] I have added a library.json file if a new loader/tool was added
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I ran `make format; make lint` to appease the lint gods

### CODE REVIEW

**Review by jerryjliu — APPROVED** [review]
looks good! if you had the chance would love to also have an integration in the main repo

File: llama_hub/llama_packs/ollama_query_engine/base.py [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+        """Get modules."""
+        return {"llm": self.llm, "index": self.index}
+
+    def run(self, query_str: str, **kwargs: Any) -> Any:
+        """Run the pipeline."""
+        query_engine = self.index.as_query_engine(**kwargs)
+        return query_engine.query(query_str)
+
+
+class OllamaEmbedding(BaseEmbedding):
```
[the remark this answers is not in the document]
Comment (chnsagitchen) [reply llama_hub/llama_packs/ollama_query_engine/base.py:46]: @jerryjliu I think you mean OllamaEmbedding class. I will prepare the PR for llama-index !

**Comment by jerryjliu** [comment]
@chnsagitchen do you have a twitter/linkedin? (feel free to dm me e.g. on Discord). would love to feature this

**Comment by chnsagitchen** [comment]
> @chnsagitchen do you have a twitter/linkedin? (feel free to dm me e.g. on Discord). would love to feature this

Hi @jerryjliu , thanks for asking. I am based on not using twitter/linkedin quite often due to some reasons ... Feel free to refer me with github account.

**Comment by jerryjliu** [comment]
no problem, thanks for letting me know

**Pull request merged**